GitHub.Collectors/PointCollectorInput.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.CloudMine.Core.Collectors.Error;
+using Microsoft.CloudMine.GitHub.Collectors.Model;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace Microsoft.CloudMine.GitHub.Collectors
+{
+    [Serializable]
+    public class PointCollectorInput
+    {
+        public string Url { get; set; }
+        public virtual string RecordType { get; set; }
+        public virtual string ApiName { get; set; }
+        public JObject Context { get; set; }
```

Comment:
Can we make Context similar to what we have in ADO point collector, a dictionary of <string, JToken>?